Tests/OctoKitTests/PullRequestTests.swift

```diff
@@ -21,6 +21,7 @@ class PullRequestTests: XCTestCase {
                 XCTAssertEqual(pullRequests.id, 1)
                 XCTAssertEqual(pullRequests.title, "new-feature")
                 XCTAssertEqual(pullRequests.body, "Please pull these awesome changes")
+                XCTAssertEqual(pullRequests.labels?.count ?? 0, 1)
```

Comment:
Is the default value needed?

```suggestion
                XCTAssertEqual(pullRequests.labels?.count, 1)
```

Reply:
ah in the test probably not!